x-pack/plugins/cloud_security_posture/server/routes/status/status.ts

```diff
@@ -148,35 +209,62 @@ const getCspStatus = async ({
     },
   ];
 
-  const status = calculateCspStatusCode(
+  const statusCspm = calculateCspStatusCode(
+    POSTURE_TYPE_CSPM,
+    {
+      findingsLatest: findingsLatestIndexStatusCspm,
+      findings: findingsIndexStatusCspm,
+      score: scoreIndexStatusCspm,
+    },
+    installedPackagePoliciesTotalCspm,
+    healthyAgentsCspm,
+    calculateDiffFromNowInMinutes(installation?.install_started_at || MIN_DATE),
+    installedPolicyTemplates
+  );
+
+  const statusKspm = calculateCspStatusCode(
+    POSTURE_TYPE_KSPM,
     {
-      findingsLatest: findingsLatestIndexStatus,
-      findings: findingsIndexStatus,
-      score: scoreIndexStatus,
+      findingsLatest: findingsLatestIndexStatusKspm,
+      findings: findingsIndexStatusKspm,
+      score: scoreIndexStatusKspm,
     },
-    installedPackagePoliciesTotal,
-    healthyAgents,
-    calculateDiffFromNowInMinutes(installation?.install_started_at || MIN_DATE)
+    installedPackagePoliciesTotalKspm,
+    healthyAgentsKspm,
+    calculateDiffFromNowInMinutes(installation?.install_started_at || MIN_DATE),
+    installedPolicyTemplates
   );
 
-  if (status === 'not-installed')
+  if ((statusCspm && statusKspm) === 'not-installed')
     return {
-      status,
+      cspm: {
+        status: statusCspm,
+        healthyAgents: healthyAgentsCspm,
+        installedPackagePolicies: installedPackagePoliciesTotalCspm,
+      },
+      kspm: {
+        status: statusKspm,
+        healthyAgents: healthyAgentsKspm,
+        installedPackagePolicies: installedPackagePoliciesTotalKspm,
+      },
       indicesDetails,
       latestPackageVersion: latestCspPackageVersion,
-      installedPolicyTemplates,
-      healthyAgents,
-      installedPackagePolicies: installedPackagePoliciesTotal,
       isPluginInitialized: isPluginInitialized(),
     };
 
   const response = {
-    status,
+    cspm: {
+      status: statusCspm,
+      healthyAgents: healthyAgentsCspm,
+      installedPackagePolicies: installedPackagePoliciesTotalCspm,
+    },
+    kspm: {
+      status: statusKspm,
+      healthyAgents: healthyAgentsKspm,
+      installedPackagePolicies: installedPackagePoliciesTotalKspm,
+    },
```

Comment:
I wonder if we could create a property to store the different policy templates. 
```
const response = {
...,
policyTemplates:  {
  cspm: {
      status: statusCspm,
      healthyAgents: healthyAgentsCspm,
      installedPackagePolicies: installedPackagePoliciesTotalCspm,
    },
    kspm: {
      status: statusKspm,
      healthyAgents: healthyAgentsKspm,
      installedPackagePolicies: installedPackagePoliciesTotalKspm,
    }
}
```

Reply:
I changed the checks above on my latest push, so I don't think the example above are still valid ? but I do agree on the readability part though maybe we can discuss about this tomorrow ?